### DISCUSSION TITLE
Bad Request (400) on adding new domain from hostinger on vercel hosting

### DISCUSSION THREAD

**Discussion by pushprajSinghPawar** [vercel/community#2801] [Help] [answered]
### Summary

I have purchased domain from hostinger after adding the domain in the domain and changing the nameserver of hostinger to
`
ns1.vercel-dns.com  ns2.vercel-dns.com
`
i validated the configuration but when i look the site it gives Bad Request (400) error .

pushpblogs.fun Valid Configuration

Redirects to www.pushpblogs.fun www.pushpblogs.fun

Production Valid Configuration Assigned to main

i waited for the dns to apply for 24 hours but still no change made.

the free domains provided by the vercel works fine but hostinger domain throws the error . 

How to see the problem here. 

### Example

https://blog-ten-zeta-33.vercel.app/ this is working but https://www.pushpblogs.fun/ is not 

### Steps to Reproduce

hostinger also gives a secret key but i am not if i have to aplly that secret key to my django application to resolve this error.

**Comment by smaeda-ks** [comment, marked as answer]
@pushprajSinghPawar you can always debug logs via `Logs` dashboard:
https://vercel.com/docs/concepts/observability/runtime-logs

In you case, the error is obvious:

> Invalid HTTP_HOST header: 'www.pushpblogs.fun'. You may need to add 'www.pushpblogs.fun' to ALLOWED_HOSTS.

**Reply by pushprajSinghPawar** [reply]
thanks  problem solved, earlier i was trying ALLOWED_HOSTS = ['127.0.0.1', '.vercel.app', 'pushpblogs.fun']
